src/main/java/mif/vu/ikeea/Controller/UserController.java

```diff
@@ -72,4 +72,22 @@
 
         return user;
     }
+
+    @PutMapping(path = "/updatePassword/{id}")
```

Comment:
Kadnagi bendrai šis controlleris atrodo /api/user/....., tai turėtų būti taip tavo route /api/user/{id}/update-password (camel case nenaudojame) ir id prieš funkciją ką nori daryti. Nes konvencija eina taip - /api/{resurso pavadinimias}/{resurso id}/{ką tam resursui nori padaryti}